docs/docs/concepts/test-suites.md

```diff
@@ -0,0 +1,60 @@
+
+## What is a Test Suite?
+A test suite is defined as a group of steps that are executed in the defined order and can access information exported by previous step executions. Each step is a test.
+
+## Chaining Tests
+The main benefit of using test suites is to chain tests together and use values obtained from a test in a subsequent test.
+
+### How Values are Shared by Tests
+When a test suite is run, a context object is created with information about that specific run. One of those pieces of information is a `variable set` object, which is empty by default. If the test suite is run when referencing an [variable set](./variable-sets.md), all values from the selected variable sets will be copied to the `variable set` object.
+
+When a test is executed within a test suite, if it generates any outputs, its outputs will be injected into the test suite context variable set object. After the outputs are injected, all subsequent tests to be run within the test suite will be able to reference those values.
+
+> :information_source: Outputs generated by steps don't modify the selected [variable set](./variable-sets.md). It only modifies the test suite run context object.
+
+Consider you have 3 tests within a test suite: A, B, and C. Tests A and B generate outputs called A_OUTPUT and B_OUTPUT, respectively. When running the test suite, we provide a variable set which contains a `HOST` variable. The execution of test A would only be able to reference `env:HOST`. B would be able to reference `env:HOST`, and `env:A_OUTPUT`. While C would be able to reference all three variables: `env:HOST`, `env:A_OUTPUT`, `env:B_OUTPUT`.
+
+> :information_source: A single test can contain as many outputs as you like.
+
+### Exposing Values from a Test to Other Tests
+Since version v0.8, Tracetest allows tests to declare `outputs`. An output is a value that is extracted from a trace by providing a [selector](./selectors) to choose which spans to use to get the information from, and an [expression](./expressions) to get the value from the selected spans. For example, consider that you want to expose the time a specific job was taken from a queue and began executing. An output would look something like the following:
+
+```yaml
+outputs:
+    - name: TIME_CANCEL_SUBSCRIPTION_MESSAGE_OBTAINED
+      selector: span[name = "Process request from cancel subscription queue"]
+      expression: attr:tracetest.time.start
+```
+
+This would create an output called `TIME_CANCEL_SUBSCRIPTION_MESSAGE_OBTAINED` that is obtained by reading the attribute `tracetest.time.start` from the span with `name` equal to `Process request from cancel subscription queue`. This value would then be injected into the variables of that test suite to be accessed by other tests within the same test suite run.
```

Comment:
```suggestion
This would create an output called `TIME_CANCEL_SUBSCRIPTION_MESSAGE_OBTAINED` that is obtained by reading the attribute `tracetest.time.start` from the span with `name` equal to `Process request from cancel subscription queue`. This value would then be injected into the variables of that Test Suite to be accessed by other tests within the same Test Suite run.
```